src/resultsdisassemblypage.cpp

```diff
@@ -245,3 +235,30 @@ void ResultsDisassemblyPage::setCostsMap(const Data::CallerCalleeResults& caller
 {
     m_callerCalleeResults = callerCalleeResults;
 }
+
+void ResultsDisassemblyPage::objdumpParse(DisassemblyOutput& disassemblyOutput)
+{
+    QVector<DisassemblyOutput::DisassemblyLine> disassemblyLines;
```

Comment:
this is the wrong place for this code, this parsing should be done within the DisassemblyOutput

i.e. you parse the raw `QByteArray output` and only return the parsed `QVector<DisassemblyOutput::DisassemblyLine>` then. The raw output isn't needed afterwards anymore

Reply:
i.e. this should happen within `DisassemblyOutput fromProcess(...)` already